src/main/resources/config/application.properties

```diff
@@ -4,18 +4,31 @@ server.port=${PORT:8080}
 management.endpoint.health.enabled=true
 management.endpoints.enabled-by-default=true
 
+
+appeal.email.from=${EMAIL_FROM:sscs@hmcts.net}
```

Comment:
I removed this email last week - did you mean to put back? Let's discuss if you disagree with me.